restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java

```diff
@@ -92,7 +92,7 @@ public void getInsertStatement() {
     values[idx++] = true; // bool_value
     assertThat(idx).isEqualTo(values.length);
 
-    BoundQuery query = documentDB.getInsertStatement("keyspace", "table", 1, values);
+    BoundQuery query = documentDB.getInsertStatement("keyspace", "table", 1, values, null);
```

Comment:
can we have one test with TTL please? I also see that we don't have it in other unit tests? Can we add?